### PR TITLE
:memo:(documentation) update outdated information

### PR DESCRIPTION
**Pull request by NamFra**
The following information is outdated and needs to be updated in the documentation:
- Contact information to access an LTI passport or a production Tenant
- Director of France Université Numérique
- BigBlueButton version

### CODE REVIEW

File: guide/creer-votre-classe-virtuelle/depuis-le-moodle-de-votre-etablissement.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Vous pouvez accéder à la solution de classes virtuelles via l’une des modali
 * **En utilisant l'activité de classes virtuelles disponible dans le cours Moodle via le standard LTI[^1]**, vous pourrez bénéficier de fonctionnalités de planification, d'invitation, d'enregistrement, d'import de documents et de paramétrage en amont. Toutes ces fonctionnalités sont documentées à l'étape suivante [configurer-votre-classe-virtuelle.md](../configurer-votre-classe-virtuelle.md "mention").
 
 {% hint style="info" %}
-Pour installer le passeport LTI ou le plugin BBB permettant l'accès à la solution dans Moodle, votre DSI peut contacter l'équipe technique de FUN en utilisant [ce formulaire de demande](https://www.fun-mooc.help/hc/fr/requests/new?ticket\_form\_id=9122250595357) ou en écrivant à l'adresse suivante : fun-ops\[a]fun-mooc.fr
+Pour installer le passeport LTI ou le plugin BBB permettant l'accès à la solution dans Moodle, votre DSI peut contacter l'équipe technique de FUN en utilisant [ce formulaire de demande](https://www.fun-mooc.help/hc/fr/requests/new?ticket_form_id=9122250595357).
```
Comment (lunika) [inline guide/creer-votre-classe-virtuelle/depuis-le-moodle-de-votre-etablissement.md:17]: No need anymore tu use `\` here ?

File: pour-les-dsi/installer-bigbluebutton-avec-esup-pod.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Près d’une cinquantaine d’établissement ont déjà déployé cette solutio
 ## Configurer l'application BigBlueButton
 
 {% hint style="warning" %}
-Pour installer la version de BBB proposée dans le cadre du Plan de relance, vous devez obtenir les identifiants du serveur BigBlueButton auprès de France Université Numérique en utilisant [ce formulaire de demande](https://www.fun-mooc.help/hc/fr/requests/new?ticket\_form\_id=9122250595357) ou en écrivant à l'adresse suivante : fun-ops\[a]fun-mooc.fr
+Pour installer la version de BBB proposée dans le cadre du Plan de relance, vous devez obtenir les identifiants du serveur BigBlueButton auprès de France Université Numérique en utilisant [ce formulaire de demande](https://www.fun-mooc.help/hc/fr/requests/new?ticket_form_id=9122250595357) (choisissez l'option "Demande de Tenant").
```
Comment (lunika) [inline pour-les-dsi/installer-bigbluebutton-avec-esup-pod.md:16]: Same here

File: pour-les-dsi/integrer-le-plugin-bbb-sur-moodle.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ description: Installez ou activez le plugin BBB dans le Moodle de votre établis
 ## **Prérequis**
 
 {% hint style="warning" %}
-Avant de vous rendre sur Moodle, vous devez obtenir les identifiants du serveur BigBlueButton auprès de France Université Numérique en utilisant [ce formulaire de demande](https://www.fun-mooc.help/hc/fr/requests/new?ticket\_form\_id=9122250595357) ou en écrivant à l'adresse suivante : fun-ops\[a]fun-mooc.fr.
+Avant de vous rendre sur Moodle, vous devez obtenir les identifiants du serveur BigBlueButton auprès de France Université Numérique en utilisant [ce formulaire de demande](https://www.fun-mooc.help/hc/fr/requests/new?ticket_form_id=9122250595357) (choisissez l'option "Demande de Tenant").
```
Comment (lunika) [inline pour-les-dsi/integrer-le-plugin-bbb-sur-moodle.md:12]: Same here

File: guide/creer-votre-classe-virtuelle/depuis-le-moodle-de-votre-etablissement.md (hunk 1 above)
Comment (NamFra) [reply guide/creer-votre-classe-virtuelle/depuis-le-moodle-de-votre-etablissement.md:17]: I thought that escaping was occurring when using the Gitbook UI. I just tested it in the preview, and it doesn't seem necessary to keep it

**Pull request merged**